src/collections/blog/2022/2022-06-13-aaditya-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "My Journey as a Contributor to GSoC Intern at Layer5"
+subtitle: "Nothing but a magical experience"
+date: 2022-06-13 08:00:00 -0630
+author: Aaditya Narayan Subedy
+thumbnail: ./aaditya-subedy-layer5-intern.png
+category: Programs
+description: "Aaditya Narayan Subedy's journey from a Contributor at Layer5 to getting an GSoC internship at Layer5"
+tags:
+ - Community
+ - Projects
+ - News
+type: Blog
+published: true
+---
+import { Link } from "gatsby";
+import { BlogWrapper } from "../../Blog.style.js";
+import aadityaFirstPr from "./aaditya-first-pr.png"
+import aadityaPresenting from "./aaditya-presenting.png"
+
+
+<BlogWrapper>
+
+### Early Interest in Coding 
+I grew up with an immense interest in computers. I used to be fascinated by how programs run on the computer that makes such an impact on the people using it. This always motivated me to work in the field of computers. 
```

Comment:
```suggestion
I had a strong interest in computers growing up. I used to find it fascinating how computer programs operate because they have such a tremendous impact on their users. This has always motivated me to work in the field of computers.```